resources/qml/Completer.qml

```diff
@@ -249,6 +249,14 @@ Control {
                             text: model.roomName
                             textFormat: Text.RichText
                         }
+                        Loader {
+                            active: Settings.displayParentInSwitcher && model.roomParent !== ""
+                            sourceComponent: Label {
+                                color: model.index == popup.currentIndex ? palette.highlightedText : palette.text
+                                text: "[" + model.roomParent + "]"
+                                font.pixelSize: popup.avatarHeight * 0.5
+                            }
+                        }
```

Comment:
Could you make this font somehow special, so that it doesn't just look like a suffix to the roomname? I.e. make it smaller, bold, different color or whatever?

Reply:
Good idea